### PR TITLE
Run operator and trade services on same port

### LINKED ISSUE #613: Allow to bind operator and trade interface on the same port

### PR DESCRIPTION
**Pull request by sekulicd**
This adds option to run trade and operator services on same port if defined by config.

This closes #613 

@tiero please review.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -95,6 +95,8 @@ const (
 	//password for unlocking the wallet, if provided wallet will be unlocked
 	//automatically
 	WalletUnlockPasswordFile = "WALLET_UNLOCK_PASSWORD_FILE"
+	// RunOnOnePortKey is used to run the daemon on one port as requested by some users
```
Comment (tiero) [inline config/config.go:98]:
```suggestion
	// RunOnOnePortKey is used to run the daemon on one port 
```

File: cmd/tdexd/main.go [hunk 2]
```diff
@@ -59,6 +59,7 @@ var (
 	rescanRangeStart              = config.GetInt(config.RescanRangeStartKey)
 	rescanGapLimit                = config.GetInt(config.RescanGapLimitKey)
 	walletUnlockPasswordFile      = config.GetString(config.WalletUnlockPasswordFile)
+	runOnOnePort                  = config.GetBool(config.RunOnOnePortKey)
```
Comment (tiero) [inline cmd/tdexd/main.go:62]: what if we infer this when trade and operator port are equal?

File: config/config.go (hunk 1 above)
Comment (sekulicd) [reply config/config.go]: Done

File: cmd/tdexd/main.go (hunk 2 above)
Comment (sekulicd) [reply cmd/tdexd/main.go]: Done

**Pull request merged**